src/include/sof/lib/dai-zephyr.h

```diff
@@ -283,7 +283,7 @@ void dai_dma_position_update(struct comp_dev *dev);
 /**
  * \brief release llp slot
  */
-void dai_release_llp_slot(struct comp_dev *dev);
+void dai_release_llp_slot(struct dai_data *dd);
```

Comment:
there is a lot of things need do, so my thinking is: first finish this PR's mission, then use other PR to do other things, I have to change legacy part, I remember, otherwise, CI build will report error.